model/blocks/blockgraph.go

```diff
@@ -0,0 +1,37 @@
+package blocks
+
+import (
+	"fmt"
+	"hash/crc32"
+	"strings"
+)
+
+type BlockNode struct {
+	Block        string
+	Height       uint64
+	Parent       string
+	ParentHeight uint64
+	Miner        string
+}
+
```

Comment:
Since this model is only used for generating dot graphs all the functionality for writing them could live here too e.g.
```suggestion

type BlockGraph = []*BlockNode

func (bg BlockGraph) String() string {
	var result strings.Builder
	result.WriteString("digraph D {\n")
	for _, b := range bg {
		result.WriteString(b.String())
	}
	result.WriteString("}\n")
	return result.String()
}
```

Reply:
Fixed.